core-ui/src/components/Extensibility/ExtensibilityCreate.js

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from 'react';
+
+import { ResourceForm } from 'shared/ResourceForm';
+import { useMicrofrontendContext } from 'shared/contexts/MicrofrontendContext';
+
+import { createTemplate } from './helpers';
+import { ResourceSchema } from './ResourceSchema';
+
+export function ExtensibilityCreate({
+  formElementRef,
+  setCustomValid,
+  resourceType,
+  resourceUrl,
+  resource: createResource,
+}) {
+  const { namespaceId: namespace } = useMicrofrontendContext();
+  const api = createResource?.resource || {};
+
+  const [resource, setResource] = useState(
+    createResource?.template ||
+      createTemplate(api, namespace, createResource?.navigation?.scope),
+  );
+  //TODO filter schema based on form configuration
+  const schema = createResource?.schema;
+
+  return (
+    <ResourceForm
+      pluralKind={resourceType}
+      singularName={resourceType}
+      resource={resource}
+      setResource={setResource}
+      formElementRef={formElementRef}
+      createUrl={resourceUrl}
+      setCustomValid={setCustomValid}
+      onlyYaml={!schema}
+    >
+      <ResourceSchema
```

Comment:
this one shouldn't be displayed only in the "create" mode?

Reply:
We only display the form in create mode for now. We can add the distinction when we implement edit mode for extensibility